pytorch/training/buildspec.yml

```diff
@@ -31,12 +31,12 @@ images:
   BuildEC2CPUPTTrainPy3DockerImage:
     <<: *TRAINING_REPOSITORY
     build: &PYTORCH_CPU_TRAINING_PY3 false
-    image_size_baseline: 5000
+    image_size_baseline: 7000
```

Comment:
whats the reason for increase in the size?

Reply:
PT binary size not increased but new dependencies versions are pulled which has more size

Reply:
increase in 2000gib?

Reply:
what are these dependencies?

Reply:
There is size increase in PT binary size and it pulls new dependencies versions. Docker history shows below. 

```
New image PT2.0.1
IMAGE          CREATED       CREATED BY                                      SIZE                
<missing>      2 days ago    |1 PYTHON=python3 /bin/sh -c /opt/conda/bin/…   2.38GB      
  
Older image PT2.0.0
<missing>      3 weeks ago   |1 PYTHON=python3 /bin/sh -c /opt/conda/bin/…   1.72GB      
```

Reply:
Reduced it to 6500 now to avoid increasing it more. Right now actual size is ~6300